packages/framer-motion-3d/src/render/gestures/use-hover.ts

```diff
@@ -21,11 +21,11 @@ export function useHover(
     return {
         onPointerOver: (event: ThreeEvent<any>) => {
             visualElement.animationState?.setActive(AnimationType.Hover, true)
-            onPointerOver?.(event)
+            onPointerOver && onPointerOver(event)
         },
         onPointerOut: (event: ThreeEvent<any>) => {
             visualElement.animationState?.setActive(AnimationType.Hover, false)
-            onPointerOut?.(event)
+            onPointerOut && onPointerOut(event)
```

Comment:
Does this matter that much? I like the readability of the `?.` operator.

Reply:
Yes - it leads to substantial size savings across the package

Reply:
If you look at the output of this operator you can see why - it is quite verbosely transpiled but also in a way that doesn't allow for good minification or gzipped (lots of unique code)

![Screenshot 2023-02-15 at 11 00 31](https://user-images.githubusercontent.com/7850794/218995921-a306f8ab-f205-42a5-96c8-0634124e3397.png)

Reply:
Also chained ones can get wild

```
const a = object?.prop?.method()

const a = (_a = object === null || object === void 0 ? void 0 : object.prop) === null || _a === void 0 ? void 0 : _a.method()
```